test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -6172,6 +6172,15 @@ def forward(self, input):
         model = Log1p()
         self.run_test(model, x)
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    def test_log10(self):
+        class Log10(torch.nn.Module):
+            def forward(self, input):
+                return torch.log1p(input)
```

Comment:
This is confusing. The test name and module class say "10" but the implementation uses log1p, which uses natural log, not log10. Can you please use [torch.log10](https://pytorch.org/docs/stable/generated/torch.log10.html)?

Reply:
I think this was a mistake nice catch thank you garymm

Reply:
Fixed that was a copy-paste bug 😟